pkg/handlers/generic/mutation/extraapiservercertsans/inject.go

```diff
@@ -63,24 +66,33 @@ func (h *extraAPIServerCertSANsPatchHandler) Mutate(
 		h.variableFieldPath...,
 	)
 	if err != nil {
-		return err
+		log.Error(
+			err,
+			"failed to get cluster config variable from extraAPIServerCertSANs mutation handler",
+		)
 	}
 	if !found {
 		log.V(5).Info("Extra API server cert SANs variable not defined")
 	}
-	apiCertSANs := extraAPIServerCertSANsVar
-	if len(apiCertSANs) == 0 {
-		log.Info("No APIServerSANs to apply")
-		return nil
+	cluster, err := clusterGetter(ctx)
+	if err != nil {
+		log.Error(
+			err,
+			"failed to get cluster from extraAPIServerCertSANs mutation handler",
+		)
 	}
-
+	defaultAPICertSANs := getDefaultAPIServerSANs(cluster)
+	apiCertSANs := slices.Clone[[]string](extraAPIServerCertSANsVar)
+	apiCertSANs = append(apiCertSANs, defaultAPICertSANs...)
```

Comment:
nit: (sorry for being annoying but I'm only just becoming properly aware of the slices package):

```suggestion
	apiCertSANs := slices.Concat(extraAPIServerCertSANsVar, defaultAPICertSANs)
```